web/public/js/directives/cd-eventbrite-integration.js

```diff
@@ -0,0 +1,76 @@
+;(function() {
+  'use strict';
+
+function cdEventbriteIntegration($window, cdEventbriteService, alertService, atomicNotifyService,
+  $localStorage, $stateParams, $state, $translate, $rootScope) {
+    return {
+      restrict: 'E',
+      template:
+        '<i class="fa fa-spinner" ng-show="saving"></i>' +
```

Comment:
It's just used to display something post callback of oauth, it needs a generic url.